config/webpack.dev.config.js

```diff
@@ -24,6 +25,10 @@ module.exports = Merge.smart(commonConfig, {
         test: /\.(js|jsx)$/,
         include: [
           path.resolve(__dirname, '../src'),
+          path.resolve(__dirname, 'node_modules/camelcase-keys'),
```

Comment:
These are ES6 packages that we depend on now, so they get included in babel.  This is the same fix that frontend-auth got.

Reply:
I think I want to get rid of these dependencies by writing out the key, key map. For example: 
```{
   levelOfEducation: 'level_of_education'
}```

This could eliminate the need for snakecase and camelcase transforms as well as make our code explicit about what exists.

Reply:
I'm totally down with that.